src/main/cli/cli.c

```diff
@@ -5737,7 +5737,7 @@ static void cliTimer(char *cmdline)
 
     pch = strtok_r(NULL, " ", &saveptr);
     if (pch) {
-        int timerIndex;
+        int timerIndex = TIMER_INDEX_UNDEFINED;
```

Comment:
https://github.com/betaflight/betaflight/pull/9303/files#diff-8c2e078f00f6f1c10102a58ae3278129R5755 can be made implicit with this change.

Reply:
Well there's a final `else` to handle all other parameters so it seemed best to leave in the `"none"` test otherwise it's not clear that's an option for the command. So I suppose we could leave the
```
        } else if (strcasecmp(pch, "none") == 0) {
```
condition empty but it seems like that would be confusing. Overall there was no way `timerIndex` could get to the usage section undefined so this is really just about fooling the compile into not complaining.

Reply:
You could just put `} else if (strcasecmp(pch, "none") != 0) {` instead of the final `} else {`.

Reply:
Ok, updated. Makes it a little more difficult to understand but not a big deal I guess.